### PR TITLE
Add final ranking

### PR DESCRIPTION
**Pull request by a-pralat**
:)

### CODE REVIEW

File: mcda/plot/plot.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    :param outranking_matrix: given outranking matrix
+    :param transitive_reduction: enable transitive reduction, defaults to True
+
+    :return: Graph created from outranking matrix if possible,
+    otherwise returns string as a warning.
+    """
+    graph = outranking_to_graph(outranking_matrix, transitive_reduction)
+
+    if isinstance(graph, str):
+        return "Directed Acyclic Graph required for transitive_reduction"
```
Comment (agak33) [inline mcda/plot/plot.py:63]: Wouldn't it be better if we return `None` when cannot build a graph? Here about [warnings](https://docs.python.org/3.8/library/warnings.html)

File: tests/test_distillation.py [hunk 2]
```diff
@@ -76,9 +82,97 @@ def test_distillation(credibility_matrix) -> None:
...
+@pytest.fixture
+def upward_order() -> pd.Series:
+    return pd.Series([["FRA"], ["GER"], ["ITA"], ["BEL"], ["AUT"]])
+
+
+def test_order_to_outranking_matrix(
+    downward_order, downward_order_matrix, upward_order, upward_order_matrix
+) -> None:
+    assert order_to_outranking_matrix(downward_order).equals(downward_order_matrix)
+    assert order_to_outranking_matrix(upward_order).equals(upward_order_matrix)
```
Comment (agak33) [inline tests/test_distillation.py:139]: These should be two separate tests

**Review by agak33 — CHANGES_REQUESTED** [review]

File: mcda/plot/plot.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    :param transitive_reduction: enable transitive reduction, defaults to True
+
+    :return: Graph created from outranking matrix if possible,
+    otherwise returns None.
+    """
+    graph = outranking_to_graph(outranking_matrix, transitive_reduction)
+    if graph is not None:
+        return _networkx_graph_to_graphviz(graph)
+
+    return graph
```
Comment (agak33) [inline mcda/plot/plot.py:66]: this return is unnecessary

**Review by agak33 — CHANGES_REQUESTED** [review]

File: mcda/plot/plot.py (hunk 3 above)
Comment (a-pralat) [reply mcda/plot/plot.py:66]: X D ...

**Review by agak33 — APPROVED** [review]
OMG !!!!

**Review by PatrykHub — APPROVED** [review]

**Pull request merged**